server/models/migrations/next/migrate.sql

```diff
@@ -8,4 +8,34 @@ INSERT INTO unit VALUES
 INSERT INTO flux VALUES 
   (14, 'STOCK_FLUX.INVENTORY_RESET'),
   (15, 'STOCK_FLUX.INVENTORY_ADJUSTMENT');
-  
\ No newline at end of file
+  
+/* migrate next */
+
+-- @jeremielodi 23-04-2020
+-- stock consumption graph report
+
+INSERT INTO unit VALUES
```

Comment:
The only thing that should be in this file is the `INSERT INTO flux` from @mbayopanda and your additions:
```sql
INSERT INTO unit VALUES
  (270, 'compare invoiced to received','TREE.COMPARE_INVOICED_RECEIVED','Compare invoiced items to received stock', 144,'/modules/reports/invoicedReceivedStock','/reports/invoicedReceivedStock');

INSERT INTO `report` (`report_key`, `title_key`) VALUES
  ('invoicedReceivedStock', 'REPORT.COMPARE_INVOICED_RECEIVED.TITLE');
```

Please make these changes.